esp32/README.md

```diff
@@ -91,6 +91,11 @@ $ git submodule init lib/berkeley-db-1.xx
 $ git submodule update
 ```
 
+You will also need to have PySerial installed if it isn't already.
+```bash
+$ pip install pyserial
+```
+
```

Comment:
If you do a `pip install esptool`, which is what I'd recommend, it'll automatically pull pyserial in from pypi.  Ditto if you do clone the repo and `pip install -e esptool` ... 

However, the `micropython-esp32/esp32/Makefile` calls `esptool.py` by path straight from the esptool package, without installing it, so the usual python installation stuff doesn't get called and the deps don't get resolved.

Possible resolutions: 
* Add instruction to run `pip install pyserial` so that dep will be installed from pypi.
* Add instruction to run `pip install esptool` to install the whole thing from pypi
* Add instruction to run `python $ESPIDF/components/esptool_py/esptool/setup.py install` to install from local tree.